app/Constants/Invite.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Constants;
+
+class Invite
```

Comment:
@sabinshrestha8 if this class won't serve any purpose other than providing constants, use the `final` keyword to make this class unextendable